staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1372,6 +1372,14 @@ const (
 	VirtualMachineStatusWaitingForVolumeBinding VirtualMachinePrintableStatus = "WaitingForVolumeBinding"
 )
 
+// VirtualMachineStatusTransitionTimestamp gives a timestamp in relation to when a status is set on a vm
+type VirtualMachineStatusTransitionTimestamp struct {
+	//  Hold the state information of the VirtualMachine.
+	Status VirtualMachinePrintableStatus `json:"status,omitempty"`
+	// StatusTransitionTimestamp is the timestamp of when the status change occurred
+	StatusTransitionTimestamp metav1.Time `json:"statusTransitionTimestamp,omitempty"`
+}
+
```

Comment:
For printableStatus, the same status field can be encountered many times throughout the VMs lifecycle.